sequencing/clinical_annotator/dxapp.json

```diff
@@ -0,0 +1,128 @@
+      "label": "VCF File Indices",
+      "class": "array:file",
+      "optional": true,
+      "patterns": [
+        "*.vcf.gz.tbi"
+      ],
+        "help": "(Optional) Tabix index file for VCF to be annotated."
+    },
+    {
+      "name": "hgmd_pro_file",
+      "label": "HGMD PRO File",
+      "class": "file",
+      "optional": true,
+      "patterns": [
+        "*.vcf","*.vcf.gz"
+      ],
+      "help": "(Optional) Make sure the build version matches the VCF to be annotated (infomation on purchasing an HGMD PRO license can be found at: https://www.qiagenbioinformatics.com/products/human-gene-mutation-database/)."
+    },
+    {
+      "group": "Build Options",
+      "name": "build_version",
+      "label": "NCBI Build Version",
+      "class": "string",
+      "choices": ["b38","b37"],
+      "default": "b38",
+      "optional": false,
+      "help": "Please indicate which build your VCF is (the default is NCBI b38)."
+    }
+  ],
+  "outputSpec": [
+    {
+      "name": "out_variants_vcfgz",
+      "label": "VCF File(s)",
+      "class": "array:file",
+      "patterns" : ["*.vcf.gz"],
+      "help": "VCF file(s) with included annotations."
+    },
+    {
+      "name": "out_variants_vcfgztbi",
+      "label": "VCF File Indices",
+      "class": "array:file",
+      "patterns" : ["*.vcf.gz.tbi"],
+      "help": "VCF index file(s) associated with annotated VCF file(s)."
+    }
+  ],
+  "runSpec": {
+    "interpreter": "bash",
+    "distribution": "Ubuntu",
+    "release": "14.04",
+    "file": "src/clinical_annotator.sh",
+    "timeoutPolicy": {
+      "main": {
+        "hours": 24
+      },
+      "*": {
+        "hours": 8,
+        "minutes": 30
+      }
+    }
+   },	
+   "regionalOptions": {
+     "aws:us-east-1": {
+      "systemRequirements": {
+        "*": {
+          "instanceType": "mem1_ssd1_x4"
+        }
+      },
+      "assetDepends": [
+          {
+            "project": "project-FGfj3Qj035bYpjzg9y3bvXFG",
+            "folder": "/asset/",
+            "id":"record-FJ3zP8Q0K5pYbQF238891Yy2",
+            "name": "marylyn_asset",
+            "resources" : "project-FGqF5B00ByvPXG4G87yQy23G"
+          }
+       ]
+     },
+     "azure:westus": {
+      "assetDepends": [
+        {
+          "project": "project-FGp4k7j9k2Vy68736j2kBVPy",
+          "folder": "/",
+          "id": "record-FJ4058j9Vf9ZBBVB2Q7b3VQz",
+          "name": "marylyn_asset",
+          "resources" : "project-FGvp3009BJq40pkZ683v3754"
+        }
+      ],
+      "systemRequirements": {
+        "*": {
+          "instanceType": "azure:mem2_ssd1_x4"
+        }
+      }
+    }
+  },
+  "access": {
+    "project": "CONTRIBUTE"
+  },
+  "__developers__" : ["user-tnperson"],
+  "__authorizedUsers__": ["org-geisinger_affiliate", "org-geisinger", "org-support", "org-geisinger_devel"],
+  "openSource" : true,
+  "whatsNew": "updated execDepends to assetDepends and cleaned up src code;",
+  "upstreamAuthor": "Ritchie, MD"
```

Comment:
I would leave this blank or skip upstreamAuthor here as we cannot be entirely sure who (Marylyn or her lab member(s)) is the tool's author.